zap/src/main/java/org/zaproxy/zap/utils/ZapTextField.java

```diff
@@ -129,4 +130,14 @@ public void setEditsLimit(int limit) {
     public void setUndoManagerPolicy(UndoManagerPolicy policy) throws NullPointerException {
         undoManager.setUndoManagerPolicy(policy);
     }
+
+    /**
+     * Sets the prompt of this {@code ZapTextField}.
+     *
+     * @param labelText the new prompt text
+     * @since 2.14.0
+     */
+    public void setPrompt(String labelText) {
```

Comment:
Correct the name of the parameter. https://github.com/zaproxy/zaproxy/pull/8024#discussion_r1305446047